### DISCUSSION TITLE
UMC1820 setup

### DISCUSSION THREAD

**Discussion by tcross99-umc1820** [jamulussoftware/jamulus#2971] [Technical Support & Help]
I use UMC1820 as my DAW in Win10. Their ASIO drive takes control of input and output. Jamulus installs fine and sees my UMC input and I can connect  server. 

However, installation says I should only hear my input signal coming back from the server. I've plugged my headphones into my PC headphone jack but the UMC is Windows output device. I can turn the UMC monitor on the device front panel but that turns off everything including the return signal from the server.

I've tried muting both the input/output signals in the UMC control panel but it appears that my monitor signal is coming straight from the hardware and not going through the ASIO driver.

Any suggestions? Thx

**Comment by pljones** [comment]
A UMC1820  is not a DAW, it's a sound card.  You need to _not_ use it for Windows sound when using it for ASIO.  You need both your input _and your output_ devices (i.e. headphones) connected to the UMC1820 when using it for ASIO.  Start with that simple set up and see if it helps.

**Reply by tcross99-umc1820** [reply]
Thx for the response. I misspoke.You are correct. My DAW is Reaper and the UMC1820 is the default Windows sound card for the mic and output.

Both input and output devices are connected to the UMC1820.

**Reply by rdica** [reply]
Why are you using the PC headphone out?

My understanding is you're supposed to be using the interface's headphone out, and you use the interface monitoring/mix control to mix between your direct signal, and the jamulus output (presumably you set that mix 100% to jamulus as recommended).

**Comment by tcross99-umc1820** [comment]
Not using any PC interfaces (in or out) as the UMC driver redirects everything to the UMC where I use the headphone jack (there are two with separate volume controls). Main audio outputs go to monitors. I've tried using the UMC ASIO control panel to mix it's output/input.

**Comment by tcross99-umc1820** [comment]
So I knew there had to be a dumb*** answer. Turns out the UMC1820 has a "Main Out" pot that, duh!, turns off the signal to the monitors leaving the headphones jack live. Solves the speaker problem. Upon testing with a live Jamulus session with 15-20ms latency, I believe the headphone signal is still coming directly from the UMC but with the low latency it's not a problem. Tested using my own private server with my keyboard player on a wifi connection with 30-50ms latency, we could still sync satisfactorily in headphones. 

MONITORING knob adjusts the mix between
the direct inputs and computer playback:
IN: only direct inputs (0 ms latency)
MIX:50/50 mix of inputs and playback from computer

I'll continue to play with this monitor knob see if I can determine if my headphone signal is in fact coming from the computer Jamulus session as recommended by Jamulus by connecting to a high latency server and testing.

**Comment by tcross99-umc1820** [comment]
OK turns out the monitoring knob turned full right does monitor the Jamulus signal coming from the server. Full left is the signal from the UMC so you can literally blend your headphone mix between the UMC and Jamulus signal.

**Reply by rdica** [reply]
Exactly :)
